src/threei/src/syscall_table.rs

```diff
@@ -0,0 +1,124 @@
+//! rawposix syscall dispatcher table
+//! Source of truth for syscall numbers: Linux x86_64 syscall table
+//! https://github.com/torvalds/linux/blob/v6.16-rc1/arch/x86/entry/syscalls/syscall_64.tbl
+//! https://filippo.io/linux-syscall-table/
+//! Keep these in sync with glibc's lind_syscall_num.h
+use super::threei::RawCallFunc;
+use rawposix::fs_calls::{
+    access_syscall, brk_syscall, chdir_syscall, chmod_syscall, clock_gettime_syscall,
+    close_syscall, dup2_syscall, dup3_syscall, dup_syscall, fchdir_syscall, fchmod_syscall,
+    fcntl_syscall, fdatasync_syscall, fstat_syscall, fstatfs_syscall, fsync_syscall,
+    ftruncate_syscall, futex_syscall, getcwd_syscall, getdents_syscall, ioctl_syscall,
+    link_syscall, lseek_syscall, mkdir_syscall, mmap_syscall, mprotect_syscall, munmap_syscall,
+    nanosleep_time64_syscall, open_syscall, pipe2_syscall, pipe_syscall, pread_syscall,
+    pwrite_syscall, read_syscall, readlink_syscall, readlinkat_syscall, rename_syscall,
+    rmdir_syscall, sbrk_syscall, stat_syscall, statfs_syscall, sync_file_range_syscall,
+    truncate_syscall, unlink_syscall, unlinkat_syscall, write_syscall, writev_syscall,
+};
+use rawposix::net_calls::{
+    accept_syscall, bind_syscall, connect_syscall, epoll_create_syscall, epoll_ctl_syscall,
+    epoll_wait_syscall, gethostname_syscall, getpeername_syscall, getsockname_syscall,
+    getsockopt_syscall, listen_syscall, poll_syscall, recvfrom_syscall, select_syscall,
+    sendto_syscall, setsockopt_syscall, shutdown_syscall, socket_syscall, socketpair_syscall,
+};
+use rawposix::sys_calls::{
+    exec_syscall, exit_syscall, fork_syscall, getegid_syscall, geteuid_syscall, getgid_syscall,
+    getpid_syscall, getppid_syscall, getuid_syscall, kill_syscall, setitimer_syscall,
+    sigaction_syscall, sigprocmask_syscall, wait_syscall, waitpid_syscall,
+};
+
+/// According to the Linux version
+/// In glibc, waitpid() is actually implemented by calling wait4(),
+/// so the Linux kernel itself does not provide a separate syscall
+/// number for waitpid.
+/// In lind-wasm, however, we treat wait and waitpid as distinct
+/// syscalls, assigning them arbitrary syscall numbers. These are
+/// only resolved later in rawposix, where wait is internally implemented
+/// by invoking waitpid with options = 0.
+/// This design choice may become a future TODO: we could adopt a
+/// similar approach in lind-glibc by having wait() directly call
+/// waitpid(), and then remove the separate wait implementation from
+/// rawposix.
+pub const SYSCALL_TABLE: &[(u64, RawCallFunc)] = &[
```

Comment:
Why aren't the syscall values here constants that are generated from some common source.  I'm sure that Linux,  etc. have these defined as constants in their codebase.